docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1117,10 +1117,14 @@ function setVideoInputDevice() {
   const {video} = getAudioVideoInput();
 
   if (meeting) {
+    const isMuted = localMedia.cameraStream?.muted;
```

Comment:
if the old stream was only remotely muted, `localMedia.cameraStream?.muted` will return true, so then we will do a local mute on the new stream, but I guess it's OK because either way server cannot unmute a participant if he/she is locally or remotely muted, so it won't make a difference